x-pack/plugins/actions/server/actions_client.ts

```diff
@@ -729,6 +747,120 @@ export class ActionsClient {
   public isPreconfigured(connectorId: string): boolean {
     return !!this.preconfiguredActions.find((preconfigured) => preconfigured.id === connectorId);
   }
+
+  public async getGlobalExecutionLogWithAuth({
+    dateStart,
+    dateEnd,
+    filter,
+    page,
+    perPage,
+    sort,
+    namespaces,
+  }: GetGlobalExecutionLogParams): Promise<IExecutionLogResult> {
+    this.logger.debug(`getGlobalExecutionLogWithAuth(): getting global execution log`);
+
+    const authorizationTuple = {} as KueryNode;
+    try {
+      await this.authorization.ensureAuthorized('get_execution_log');
+    } catch (error) {
+      this.auditLogger?.log(
+        connectorAuditEvent({
+          action: ConnectorAuditAction.GET_GLOBAL_EXECUTION_LOG,
+        })
+      );
+      throw error;
+    }
+
+    this.auditLogger?.log(
+      connectorAuditEvent({
+        action: ConnectorAuditAction.GET_GLOBAL_EXECUTION_LOG,
+      })
+    );
+
+    const dateNow = new Date();
+    const parsedDateStart = parseDate(dateStart, 'dateStart', dateNow);
+    const parsedDateEnd = parseDate(dateEnd, 'dateEnd', dateNow);
+
+    const eventLogClient = await this.getEventLogClient();
+
+    try {
+      const aggResult = await eventLogClient.aggregateEventsWithAuthFilter(
+        'action',
+        authorizationTuple,
+        {
+          start: parsedDateStart.toISOString(),
+          end: parsedDateEnd.toISOString(),
+          aggs: getExecutionLogAggregation({
+            filter,
+            page,
+            perPage,
+            sort,
+          }),
+        },
+        namespaces
+      );
+
+      return formatExecutionLogResult(aggResult);
+    } catch (err) {
+      this.logger.debug(
+        `actionsClient.getGlobalExecutionLogWithAuth(): error searching global event log: ${err.message}`
+      );
+      throw err;
+    }
+  }
+
+  public async getGlobalExecutionKpiWithAuth({
+    dateStart,
+    dateEnd,
+    filter,
+    namespaces,
+  }: GetGlobalExecutionKPIParams) {
+    this.logger.debug(`getGlobalExecutionLogWithAuth(): getting global execution log`);
+
+    const authorizationTuple = {} as KueryNode;
+    try {
+      await this.authorization.ensureAuthorized('get_execution_kpi');
+    } catch (error) {
+      this.auditLogger?.log(
+        connectorAuditEvent({
+          action: ConnectorAuditAction.GET_GLOBAL_EXECUTION_KPI,
+        })
+      );
+      throw error;
+    }
+
+    this.auditLogger?.log(
+      connectorAuditEvent({
+        action: ConnectorAuditAction.GET_GLOBAL_EXECUTION_KPI,
+      })
+    );
+
+    const dateNow = new Date();
+    const parsedDateStart = parseDate(dateStart, 'dateStart', dateNow);
+    const parsedDateEnd = parseDate(dateEnd, 'dateEnd', dateNow);
+
+    const eventLogClient = await this.getEventLogClient();
+
+    try {
+      const aggResult = await eventLogClient.aggregateEventsWithAuthFilter(
+        'alert',
```

Comment:
should this be `action`?

Reply:
Resolved in this commit 578d4334020d003439299ed65f05994a1ecdf4c7